recipes/cryptominisat/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set name = "cryptominisat" %}
+{% set version = "5.0.1" %}
+{% set sha256 = "789d53c90ecbd5910f48e4e1bd835e374b268e0b3872a81fd5c6178a30a5e263" %}
+{% set author = "msoos" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/{{ author }}/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: False
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+
```

Comment:
You need some tests. See:

https://conda-forge.org/docs/testing.html#simple-existence-tests

or for another package that might be a useful example:

https://github.com/conda-forge/yaml-feedstock/blob/master/recipe/meta.yaml#L35-L45